packages/project-management-automation/lib/tasks/add-milestone/index.js

```diff
@@ -18,20 +17,18 @@ const DAYS_PER_RELEASE = 14;
 
 /**
  * Returns true if the given error object represents a duplicate entry error, or
- * false otherwise. The error is expected to be (though not verified explicitly
- * as) an instance of Octokit's RequestError (`@octokit/request-error`).
+ * false otherwise.
  *
- * @see https://github.com/octokit/rest.js/issues/684
- * @see https://developer.github.com/v3/#client-errors
- * @see https://github.com/octokit/request.js/blob/2a1d768/src/fetch-wrapper.ts#L79-L80
- *
- * @param {HookError} error Error to test.
+ * @param {RequestError} requestError Error to test.
  *
  * @return {boolean} Whether error is a duplicate validation request error.
  */
-const isDuplicateValidationError = ( error ) =>
-	Array.isArray( error.errors ) &&
-	error.errors.some( ( { code } ) => code === 'already_exists' );
+const isDuplicateValidationError = ( requestError ) => {
+	// The included version of RequestError provides no way to access the
+	// full 'errors' array that the github REST API returns. Hopefully they
+	// resolve this soon!
+	return requestError.message.includes( 'already_exists' );
```

Comment:
Unfortunately `@actions/github` uses quite an old version of their `RequestError` library.